src/state/mapConfig/selectors.js

```diff
@@ -0,0 +1,7 @@
+
+export function getMapTitle(state) {
+  return state.mapConfig.about.title;
+}
+export function getMapAbstract(state) {
+  return state.mapConfig.about.abstract;
```

Comment:
it is, right. should change it to `state.mapConfig.about["abstract"]`
the json returned from the geonode does return it as `abstract`